tests/ballerina-tools-integration-test/src/test/java/org/ballerinalang/test/utils/PackagingTestUtils.java

```diff
@@ -0,0 +1,99 @@
+/*
+ * Copyright (c) 2018, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.ballerinalang.test.utils;
+
+import org.testng.Assert;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.ArrayList;
+import java.util.Comparator;
+import java.util.List;
+import java.util.Locale;
+import java.util.Map;
+
+/**
+ * Util methods needed for Packaging test cases.
+ */
+public class PackagingTestUtils {
+
+    /**
+     * Delete files inside directories.
+     *
+     * @param dirPath directory path
+     * @throws IOException throw an exception if an issue occurs
+     */
+    public static void deleteFiles(Path dirPath) throws IOException {
+        Files.walk(dirPath)
+             .sorted(Comparator.reverseOrder())
+             .forEach(path -> {
+                 try {
+                     Files.delete(path);
+                 } catch (IOException e) {
+                     Assert.fail(e.getMessage(), e);
+                 }
+             });
+    }
+
+    /**
+     * Get org-name of user.
+     *
+     * @return org name
+     */
+    static String getOrgName() {
+        String orgName = System.getProperty("user.name");
```

Comment:
Lets change this logic a bit. Lets have a default value for the org name. Then we try to get the new value from system property using an optional value. And based on not null value present, then you can assign the new value to the default value.

Reply:
Removed this method since we no longer need it.